libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -3093,7 +3093,7 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
       baseChain: "goerli",
       chainId: 5, // goerli
       networkId: 5,
-      hardfork: "petersburg",
+      hardfork: "london",
```

Comment:
- Why `london` specifically and not a more recent fork ?
- Why `ethereum_ropsten` is still using `petersburg` ?

Reply:
Because EIP-1559 has been introduced with the London fork, so we're supporting up until this point.
Ropsten isn't supported anymore in the Live.